tools/local-snapshot-runner-app/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export * from "./localSnapshotRunnerApp";
```

Comment:
There is nothing to export from this file.
Instead, it would be great to export exportFile(), such that we can leverage it directly in non-command line environments (for example, in testing, or if someone wants to build a wrapper, or something else).